src/ServiceControl.Monitoring.Tests/QueueLength/QueueLengthStoreTests.cs

```diff
@@ -194,11 +194,10 @@ public void If_send_report_is_received_value_snapshot_is_taken_for_endpoints_wit
             var now = DateTime.UtcNow;
             consumer.SnapshotCurrentQueueLengthEstimations(now);
 
-            var lengths = new MonitoredEndpoint { Name = "ReceiverA" };
-            consumer.FillIn(new[] { lengths }, now);
+            var lengths = consumer.GetIntervals(now);
 
-            Assert.IsNotNull(lengths.QueueLength);
-            Assert.AreEqual(3 - 1, lengths.QueueLength.Points.Last());
+            Assert.AreEqual(1, lengths.Length);
+            Assert.AreEqual(3 - 1, lengths[0].Intervals.First().TotalValue);
```

Comment:
I'd extract `3` and `1` to consts to make `1` different from the `1` in the line 199.